docs/articles/documentation/test-api/built-in-waiting-mechanisms.md

```diff
@@ -10,9 +10,15 @@ redirect_from:
 TestCafe has built-in automatic waiting mechanisms, so that it does not need dedicated API to wait for page elements to appear or redirects to happen.
 
 This topic describes how these mechanisms work with [test actions](actions/README.md),
-[assertions](assertions/README.md), [selectors](selecting-page-elements/selectors/README.md) and navigation.
+[assertions](assertions/README.md), [selectors](selecting-page-elements/selectors/README.md), requests and navigation.
```

Comment:
[assertions](assertions/README.md), [selectors](selecting-page-elements/selectors/README.md), requests, and navigation